src/Nethermind/MyPlugin/TestPlugin.cs

```diff
@@ -0,0 +1,47 @@
+using System.Threading.Tasks;
+using Nethermind.Api;
+using Nethermind.Api.Extensions;
+using Nethermind.Core;
+using Nethermind.Pipeline;
+using Nethermind.Pipeline.Publishers;
+
+namespace MyPlugin
+{
+    public class TestPlugin : INethermindPlugin
```

Comment:
It is here just for the time of the draft to show how plugin might look like with use of Pipeline